tests/integration/cmd_storage_test.go

```diff
@@ -41,8 +41,8 @@ var _ = Describe("odo storage command", func() {
 
 	Context("when running storage command without required flag(s)", func() {
 		It("should fail", func() {
-			helper.CopyExample(filepath.Join("source", "dotnet"), context)
-			helper.CmdShouldPass("odo", "component", "create", "dotnet", "dotnet", "--app", "dotnetapp", "--project", project, "--context", context)
+			helper.CopyExample(filepath.Join("source", "openjdk"), context)
```

Comment:
Need import java IS ```oc.ImportJavaIs(project)```. Why dotnet component is removed ?

Reply:
Why is the dotnet test replaced with java:8?

Reply:
This test is never creating the component. So it doesn't really matter.
DotNet image has a lot of problems, and it is not our primary focus right now.

Reply:
Hmm, understandable.